src/components/SearchInput/SearchInput.tsx

```diff
@@ -8,11 +9,21 @@ type Props = {
   value: string;
   setValue: (e: any) => void;
   onKeyDown: (e: any) => void;
+  onInput?: (e: FormEvent<HTMLInputElement>) => void;
   className?: string;
   placeholder?: string;
+  autoFocus?: boolean;
 };
 
-export default function SearchInput({ value, setValue, onKeyDown, className, placeholder }: Props) {
+export default function SearchInput({
+  value,
+  setValue,
+  onKeyDown,
```

Comment:
if i'm checking correctly onKeyDown is not used currently, empty handler is passed to the SearchInput. is there a reason to have this handler?